torchvision/transforms/transforms.py

```diff
@@ -819,13 +813,7 @@ def __init__(self, transformation_matrix, mean_vector):
         self.mean_vector = mean_vector
 
     def __call__(self, tensor):
-        """
-        Args:
```

Comment:
So you think it's better to remove those parts of the docstrings? It still carry some useful information, like the expected shapes of the tensors.

Reply:
You're right, the shape might be of interest.

Reply:
Should be addressed in b9a76785c382fac762827ba5a945d03cd56c3711